imsim/batoid_wcs.py

```diff
@@ -31,6 +32,45 @@
 #     Pixels on an individual detector.
 
 
+def focal_to_pixel(fpx, fpy, det):
+    """
+    Parameters
+    ----------
+    fpx, fpy : array
+        Focal plane position in millimeters in DVCS
+        See https://lse-349.lsst.io/
+    det : lsst.afw.cameraGeom.Detector
+        Detector of interest.
+
+    Returns
+    -------
+    x, y : array
+        Pixel coordinates.
+    """
+    tx = det.getTransform(cameraGeom.FOCAL_PLANE, cameraGeom.PIXELS)
+    x, y = tx.getMapping().applyForward(np.vstack((fpx, fpy)))
+    return x.ravel(), y.ravel()
+
+
+def pixel_to_focal(x, y, det):
```

Comment:
These two functions probably ought to move to your new batoid_utils file.

Reply:
I did not move them, because they are not batoid specific and I did not want to initiate a big module messed up with tons of unrelated things in it. I like the idea of grouping things together which also change together (it is a coordinate transform which I think fits well into "wcs").
Of course we also could argue about the new `batoid_utils` module. I am also open here.

Reply:
OK.  It seems to me that these are both utility functions used by multiple batoid-related modules, so would make sense to be in `batoid_utils`.  Or possibly in a non-batoid-specific `utils`.  Seems cleaner to me than having `photon_ops` load some things from `batoid_utils` and some from `batoid_wcs`.  
But I don't care that much if you prefer to keep them here.

Reply:
created a new `utils` module.